palette/Cargo.toml

```diff
@@ -67,10 +68,14 @@ version = "1"
 optional = true
 
 [dependencies.wide]
-version = "0.7.3"
+version = "0.7"
 optional = true
 default-features = false
 
+[dependencies.simba]
+version = "0.9"
```

Comment:
It would be good to specify a minor version here too.

Reply:
Done 👍